patternfly/patternfly-elements
# pfe-card -- fix storybook issue with incorrect attribute

### What has changed and why
In Storybook, the incorrect attribute of color is shown, instead of pfe-color
![image](https://user-images.githubusercontent.com/456863/67040248-16188100-f0f1-11e9-9a18-97f3c3054cf6.png)

https://patternfly.github.io/patternfly-elements/demo/

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1.  `npm run storybook`
2. view card, ensure `pfe-color` is shown
3. Ensure color property changes still work with the storybook widget

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
